### DISCUSSION TITLE
Add upload button for favicon of Prowlarr providers

### DISCUSSION THREAD

**Discussion by Rouzax** [pymedusa/Medusa#10059] [Ideas & Feature requests]
Would be really handy if there is an option to upload a favicon when adding a Prowlarr provider/
On this page
![image](https://user-images.githubusercontent.com/4103090/140898869-6e7299b0-a846-4aed-a6c9-0f3277b588f9.png)

Have also added it to Prowlarr as request for them to send the image downstream
https://github.com/Prowlarr/Prowlarr/issues/589

**Comment by neo-neo1** [comment]
This request already exists:

- https://github.com/pymedusa/Medusa/discussions/9893
